libnestutil/propagator_stability.h

```diff
@@ -0,0 +1,29 @@
+/*
+ *  propagator_stability.h
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef PROPAGATOR_STABILITY_H
+#define PROPAGATOR_STABILITY_H
+
+double propagator_31( double tau_syn, double tau, double C, double h );
+double propagator_32( double tau_syn, double tau, double C, double h );
```

Comment:
@ingablundell: Do we have to generate this as well if the dimensionality of the matrix changes? Are `_31` and `_32` good names or is there a more generic name? At least we should be aware that this might be something to consider in the context of NESTML.